docs/contributing.rst

```diff
@@ -255,7 +255,74 @@ To issue an unstable version when the current version is stable, specify the new
 version explicitly, like ``make release bump="--new-version 4.0.0-alpha.1 devnum"``.
 
 
+Generating new fixtures
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Our integration tests make use of Geth and Parity/OpenEthereum private networks.
+When new versions of the client software are introduced, new fixtures should be
+generated.
+
+.. note::
+
+    A "fixture" is a pre-synced network. It's the result of configuring and running
+    a client, deploying the test contracts, and saving the resulting state for
+    testing Web3.py functionality against.
+
+
+Geth fixtures
+^^^^^^^^^^^^^
+
+1. Install the desired Geth version on your machine locally. The Geth team only
+   explicitly supports the current version of their client at any given point,
+   so older versions are best installed via `py-geth`_.
```

Comment:
IIRC, CI builds from py-geth (not 100% sure on that), so might be worth mentioning that py-geth may need to be updated to build the latest and greatest version.